UnityAds/UnityRouter.m

```diff
@@ -33,13 +33,20 @@ @interface UnityRouter ()
 
 @implementation UnityRouter
 
+static int impressionOrdinal = 0;
```

Comment:
Is UnityRouter only ever instantiated once? Or is it possible for multiple routers?